$DragonFly: src/secure/lib/libssh/Attic/openbsd-compat,port-tun.c.patch,v 1.2 2006/09/28 18:42:50 corecode Exp $
--- port-tun.c	2006-09-02 07:32:40.000000000 +0200
+++ port-tun.c	2006-09-28 19:39:51.000000000 +0200
@@ -110,7 +110,7 @@
 #include <net/if.h>
 
 #ifdef HAVE_NET_IF_TUN_H
-#include <net/if_tun.h>
+#include <net/tun/if_tun.h>
 #endif
 
 int
